@main def runtimeError = throw RuntimeException()
